telemetry/definitions/commonDefinitions.json

```diff
@@ -2238,6 +2243,10 @@
                 {
                     "type": "reason",
                     "required": false
+                },
+                {
+                    "type": "locale",
```

Comment:
What do you think about making this an optional field at the code generator level, that way all metrics have the option of emitting this?

Reply:
ah i like that idea - will make that adjustment